src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -200,77 +200,77 @@ populate_aie_core(boost::property_tree::ptree _pt,boost::property_tree::ptree& t
     tile.put("row", pt.get<uint32_t>("row"));
 
     std::string status;
-    for (auto& node: pt.get_child("core.Status", empty_pt)) {
+    for (auto& node: pt.get_child("core.status", empty_pt)) {
       status +=(status.empty()?"":", ")+node.second.data();
     }
     if(!status.empty())
       tile.put("core.status", status);
 
-    for (auto& node: pt.get_child("core.PC", empty_pt))
+    for (auto& node: pt.get_child("core.pc", empty_pt))
       tile.put("core.program_counter", node.second.data());
 
-    for (auto& node: pt.get_child("core.LR", empty_pt))
+    for (auto& node: pt.get_child("core.lr", empty_pt))
       tile.put("core.link_register", node.second.data());
 
-    for (auto& node: pt.get_child("core.SP", empty_pt))
+    for (auto& node: pt.get_child("core.sp", empty_pt))
       tile.put("core.stack_pointer", node.second.data());
-
+/*
     std::string mode;
-    for (auto& node: pt.get_child("dma.Mode.MM2S", empty_pt)) {
+    for (auto& node: pt.get_child("dma.mode.mm2s", empty_pt)) {
       mode += (mode.empty()?"":", ")+node.second.data();
     }
 
     if(!mode.empty())
       tile.put("dma.mm2s.mode", mode);
 
     mode.clear();
-    for (auto& node: pt.get_child("dma.Mode.S2MM", empty_pt)) {
+    for (auto& node: pt.get_child("dma.mode.s2mm", empty_pt)) {
       mode += (mode.empty()?"":", ")+node.second.data();
     }
 
     if(!mode.empty())
       tile.put("dma.s2mm.mode", mode);
-
+*/
     boost::property_tree::ptree mm2s_array;
-    auto queue_size = pt.get_child("dma.Queue_size.MM2S").begin();
-    auto queue_status = pt.get_child("dma.Queue_status.MM2S").begin();
-    auto current_bd = pt.get_child("dma.Current_BD.MM2S").begin();
-    auto lock_id = pt.get_child("dma.Lock_ID.MM2S").begin();
+    auto queue_size = pt.get_child("dma.queue_size.mm2s").begin();
+    auto queue_status = pt.get_child("dma.queue_status.mm2s").begin();
+    auto current_bd = pt.get_child("dma.current_bd.mm2s").begin();
+    //auto lock_id = pt.get_child("dma.lock_id.mm2s").begin();

```

Comment:
Remove code

Reply:
according to aie driver spec these nodes were that but in actual implementation they remove but these changes will come in future. so commented this part of code.
But now i am removing.